srai/embedders/base.py

```diff
@@ -0,0 +1,22 @@
+"""Base class for embedders."""
+
+import abc
+from typing import Any
+
+import geopandas as gpd
+
+
+class BaseEmbedder(abc.ABC):
+    """Base abstract class for embedders."""
+
+    @abc.abstractmethod
+    def embed(self, gdf: gpd.GeoDataFrame, **kwargs: Any) -> gpd.GeoDataFrame:
```

Comment:
will it be easy to store the embedding results, e.g. numpy arrays or other vectors in GeoDataFrames?

Reply:
Well, I was thinking about it and it makes sense, what you said. Returning numpy array seems like a better choice, I will change it